x-pack/plugins/secops/public/components/page/network/ip_overview/field_renderers.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { getOr, isString } from 'lodash/fp';
+import React from 'react';
+
+import {
+  AutonomousSystem,
+  HostEcsFields,
+  IpOverviewData,
+  IpOverviewType,
+  Overview,
+} from '../../../../graphql/types';
+import { DefaultDraggable } from '../../../draggables';
+import { getEmptyTagValue } from '../../../empty_value';
+import { ExternalLinkIcon } from '../../../external_link_icon';
+import { PreferenceFormattedDate } from '../../../formatted_date';
+import { HostDetailsLink, ReputationLink, VirusTotalLink, WhoIsLink } from '../../../links';
+
+import { IpOverviewId } from './index';
+import * as i18n from './translations';
+
+export const locationRenderer = (
+  fieldNames: string[],
+  data: IpOverviewData
+): React.ReactElement => {
+  return fieldNames.length > 0 &&
+    fieldNames.every((fieldName: string) => getOr(null, fieldName, data)) ? (
+    <EuiFlexGroup alignItems="center" gutterSize="none" data-test-subj="location-field">
+      {fieldNames.map((fieldName: string, index: number) => {
+        const locationValue = getOr('', fieldName, data);
+        return (
+          <React.Fragment key={`${IpOverviewId}-${fieldName}`}>
+            {index ? ',\u00A0' : ''}
+            <EuiFlexItem grow={false}>
+              <DefaultDraggable
+                id={`${IpOverviewId}-${fieldName}`}
+                field={fieldName}
+                value={locationValue}
+              />
+            </EuiFlexItem>
+          </React.Fragment>
+        );
+      })}
+    </EuiFlexGroup>
+  ) : (
+    getEmptyTagValue()
+  );
+};
+
+export const dateRenderer = (fieldName: string, data: Overview): React.ReactElement => {
+  const value = getOr(null, fieldName, data);
+  return value && isString(value) ? (
+    <PreferenceFormattedDate value={new Date(value)} />
```

Comment:
Thanks for the heads up! I've switched to use `FormattedDate` in the `dateRenderer`. I kept the renderer even though it essentially returns `FormattedDate` so that the default date component is more easily discoverable (among the other field renderers).